packages/api/src/routers/svc/newsletters.ts

```diff
@@ -96,7 +96,12 @@ export function newsletterServiceRouter() {
 
       const result = await saveNewsletterEmail(data)
       if (!result) {
-        console.log('Error creating newsletter link from data', data)
+        console.log(
+          'Error creating newsletter link from data',
+          data.email,
```

Comment:
This causes some lint error

Reply:
I guess we need to define the data structure

Reply:
yeah, kind of annoying because this is right above 

```
if (
        !('email' in data) ||
        !('content' in data) ||
        !('title' in data) ||
        !('author' in data)
      ) {
```